src/hardware/targets.json

```diff
@@ -158,6 +158,12 @@
                 "lua_name": "DupleTX",
                 "layout_file": "DIY 2400 DupleTX.json",
                 "upload_methods": ["uart", "wifi", "etx"]
+            },
+            "dupletxesp": {
+                "product_name": "Generic Full-duplex 2.4GHz ESP TX",
```

Comment:
Have 8285 in the name as an identifier would be nice.

Reply:
Renamed to `Generic ESP8285 Full-duplex 2.4GHz TX` since that matches the order of our other names like `Generic ESP8285 2.4Ghz RX`